base_layer/core/tests/helpers/block_builders.rs

```diff
@@ -114,7 +114,7 @@ pub fn _create_act_gen_block() {
     let factories = CryptoFactories::default();
     let mut header = BlockHeader::new(consensus_manager.consensus_constants(0).blockchain_version());
     let value = consensus_manager.emission_schedule().block_reward(0);
-    let (mut utxo, key, _) = create_utxo(value, &factories, None, &TariScript::default());
+    let (mut utxo, key, _) = create_utxo(value, &factories, Default::default(), &TariScript::default());
```

Comment:
```suggestion
    let (utxo, key, _) = create_utxo(value, &factories, OutputFeatures::create_coinbase(1), &TariScript::default());
```

Reply:
```suggestion
    let (mut utxo, key, _) = create_utxo(value, &factories, Default::default(), &script!(Nop));
```
We should probably do this as the default will be spendable.

Reply:
These are working tests, so are you suggesting this as a general best practise?